pygmt/helpers/testing.py

```diff
@@ -154,7 +154,7 @@ def load_static_earth_relief():
         A grid of Earth relief for internal tests.
     """
     fname = which("@static_earth_relief.nc", download="c")
-    return load_dataarray(fname)
+    return load_dataarray(fname, decode_kind="grid")
```

Comment:
Again, with `load_dataarray` deprecated, we should use `xr.load_dataarray(fname, engine="gmt", decode_kind="grid")` instead, but it can be done in a separate PR.

Reply:
Started draft PR at #3922, will cherrypick things across.